robotiq_c_model_control/action/Gripper.action

```diff
@@ -22,6 +22,11 @@ uint8 FAILURE=1
 uint8 position
 uint8 result_code
 uint8 object_detection
+
+uint8 HAS_OBJ_UNKNOWN=0
+uint8 HAS_OBJ_YES=1
+uint8 HAS_OBJ_NO=2
+uint8 has_object
```

Comment:
How is this different from `object_detection`?
Is there a risk in message mismatch from a component expecting / providing message without this field?

Reply:
Great question -- I had to look it up. So "object_detection" is the value of the register gOBJ in the robotiq status message whereas has_object is a translation of that value into a more sensible "YES/NO/UNKNOWN"

Reply:
I don't love that we're showing the raw register guts in our action response -- I might see where this is used and if unused delete the `object_detection` field all together

Reply:
it doesn't appear to be used anywhere in release-2.3 so maybe it can be purged. Thoughts from the rest of the gang?

Reply:
I'm going to propose rolling these changes into develop rather than 2.3. Created https://readyrobotics.atlassian.net/browse/RPI-31